piqe_ocp_lib/api/resources/ocp_nodes.py

```diff
@@ -166,35 +184,35 @@ def execute_command_on_a_node(self, node_name, command_to_execute):
         if ret != 0:
             # Spin a new container for the node
             container_definition = """
-{
-  "spec": {
-    "hostPID": true,
-    "hostNetwork": true,
-    "nodeSelector": { "kubernetes.io/hostname": "%s" },
-    "tolerations": [{
-      "operator": "Exists"
-    }],
-    "containers": [
-      {
-        "name": "nsenter",
-        "image": "alexeiled/nsenter:2.34",
-        "command": [
-          "/nsenter", "--all", "--target=1", "--", "su", "-"
-        ],
-        "stdin": true,
-        "tty": true,
-        "securityContext": {
-          "privileged": true
-        },
-        "resources": {
-          "requests": {
-            "cpu": "10m"
-          }
-        }
-      }
-    ]
-  }
-}""" % node_name
+                {
+                  "spec": {
+                    "hostPID": true,
+                    "hostNetwork": true,
+                    "nodeSelector": { "kubernetes.io/hostname": "%s" },
+                    "tolerations": [{
+                      "operator": "Exists"
+                    }],
+                    "containers": [
+                      {
+                        "name": "nsenter",
+                        "image": "alexeiled/nsenter:2.34",
+                        "command": [
+                          "/nsenter", "--all", "--target=1", "--", "su", "-"
+                        ],
+                        "stdin": true,
+                        "tty": true,
+                        "securityContext": {
+                          "privileged": true
+                        },
+                        "resources": {
+                          "requests": {
+                            "cpu": "10m"
+                          }
+                        }
+                      }
+                    ]
+                  }
+                }""" % node_name
             command = ("kubectl run %s --restart=Never --image "
```

Comment:
I know we have talked about this in the past, but we need to figure out how to make sure kubectl available as a dependency if we are going to be using it as part of our libs. As it is, we are not installing it or explicitly marking it as a dependency, so running this will break, correct?

Reply:
We added this method when we worked on Tigera usecase. In this PR, I only made indentation change to make it more readable and align with other code.

When we install openshift as part of setup.py, openshift will bring kubectl as part of dependency. This method was working as is  when we were using in Tigera usecase.